### DISCUSSION TITLE
Anybody managed to update settings for dart lsp?

### DISCUSSION THREAD

**Discussion by arnaud-secondlayer** [zed-industries/zed#12545] [Feature Requests]
I'm trying to set the dart line length in `settings.json` to get long lines when formatting for dart.
I tried several options but I haven't been able to get it right. Here is something I tried but doesn't work:

```
{
  "theme": "Andromeda",
  "ui_font_size": 16,
  "buffer_font_size": 12,
  "buffer_font_family": "Menlo",
  "buffer_font_features": {
    "calt": false
  },
  "format_on_save": "off",
  "terminal": { "font_size": 12, "font_family": "Menlo", "dock": "bottom" },
  "git": {
    "inline_blame": {
      "enabled": false
    }
  },
  "lsp": {
    "dart": {
      "initialization_options": {
        "dart.lineLength": 160
      }
    }
  }
}
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
